os-eap-launch/added/launch/datasource-common.sh

```diff
@@ -186,8 +186,9 @@ function generate_datasource_common() {
   fi
 
   if [ -n "$TIMER_SERVICE_DATA_STORE" -a "$TIMER_SERVICE_DATA_STORE" = "${service_name}" ]; then
+    local refreshInterval=${TIMER_SERVICE_DATA_STORE_REFRESH_INTERVAL:--1}
```

Comment:
"-1" is the default in case nothing is specified (which was the case before). See https://wildscribe.github.io/WildFly/10.1/subsystem/ejb3/service/timer-service/database-data-store/index.html. 

I am not convinced it is a good default, but wanted to make the change 100% backwards compatible.